packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -137,6 +137,8 @@
     "screenIsBeingSharedMessage": "You are sharing your screen",
     "screenShareLoadingMessage": "Loading {participant}'s screen",
     "localVideoLabel": "You",
-    "localVideoCameraSwitcherLabel": "Cycle camera"
+    "localVideoCameraSwitcherLabel": "Cycle camera",
+    "localVideoMovementLabel": "Movable Local Video Tile",
+    "localVideoMovementRole": "Dialog"
```

Comment:
Dialog is a standard aria role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role
Does it need localized?